src/makeCalculator.js

```diff
@@ -36,8 +36,41 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
```

Comment:
You can access object with `this`
```suggestion
      this.result = 0;
```